<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </item>
    <item row="1" column="1">
     <widget class="QTableWidget" name="main_table"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSave_as">
     <property name="title">
      <string>Save as</string>
     </property>
     <addaction name="save_as_csv"/>
     <addaction name="save_as_txt"/>
    </widget>
    <addaction name="New_file"/>
    <addaction name="Open_file"/>
    <addaction name="separator"/>
    <addaction name="Save"/>
    <addaction name="menuSave_as"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="Add_row"/>
    <addaction name="Remove_row"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Open_file">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="save_as_csv">
   <property name="text">
    <string>as csv</string>
   </property>
  </action>
  <action name="save_as_txt">
   <property name="text">
    <string>as txt</string>
   </property>
  </action>
  <action name="New_file">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New file</string>
   </property>
  </action>
  <action name="Add_row">
   <property name="text">
    <string>Add row</string>
   </property>
  </action>
  <action name="Remove_row">
   <property name="text">
    <string>Remove row</string>
   </property>
  </action>
  <action name="Save">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
